src/stretch/agent/manipulation/dynamem_manipulation/dynamem_manipulation.py

```diff
@@ -228,12 +231,12 @@ def move_to_joints(self, joints, gripper, mode=0):
                 target1, blocking=True, head=np.array([self.pan, self.tilt]), reliable=False
             )
 
-        # self.robot.arm_to(target_state, blocking=True)
-        # self.robot.head_to(head_tilt=self.tilt, head_pan=self.pan, blocking=True, reliable=False)
+        self.robot.arm_to(target_state, blocking=True)
+        self.robot.head_to(head_tilt=self.tilt, head_pan=self.pan, blocking=True, reliable=False)
 
-        self.robot.arm_to(
-            target_state, blocking=True, head=np.array([self.pan, self.tilt]), reliable=False
-        )
```

Comment:
@hello-cpaxton , after fixing the blocking of zmq client arm to, I still prefer this way to ensure head movement

Reply:
ok!